index.html

```diff
@@ -118,7 +118,7 @@ <h2 id="installation">
       </table>
       <p>
         There are more clients listed in the
-        <a href="https://github.com/tldr-pages/tldr/wiki/tldr-clients">tldr clients wiki page</a>.
+        <a href="https://github.com/tldr-pages/tldr#how-do-i-use-it">tldr README</a>.
```

Comment:
The wiki page [was not actually merged](https://gitter.im/tldr-pages/tldr?at=5fb6fc66bf955735eb8da63f) with the README. The page was renamed, though, so the link needs to be updated:
```suggestion
        <a href="https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients">tldr clients wiki page</a>.
```

Reply:
@SmithJohnTaylor why did you resolve this comment? The link in the diff still points to the README.

Note that the list in the README is not (yet) the most comprehensive source of information about tldr clients. Until the lists are merged into the main README (see #52 for progress on that), I believe it's better to keep this link pointing to the wiki page.

Reply:
It should be **tldr-pages clients wiki page** not ~tldr clients wiki page~:
```suggestion
        <a href="https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients">tldr-pages clients wiki page</a>.
```